protocol_rfcs/collated-string-type.md

```diff
@@ -8,12 +8,33 @@ This protocol change adds support for collated strings. It consists of three cha
 * Domain metadata with active collation version
 
 --------
+> *** Add New Section after the [Clustered Table](#clustered-table) section***
+# Collations Table Feature
 
-> ***Add a new section in front of the [Primitive Types](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#primitive-types) section.***
+To support this feature:
+* The table must have Writer Version 7. A feature name `collations` must exist in the table's `writerFeatures`.
+* The feature `domainMetadata` is required in the table protocol's `writerFeatures`. 
+
+## Reader Requirements for Collations:
+
+When Collations is supported (when the `writerFeatures` field of a table's protocol action contains `collations`), then:
+- Readers could do comparisons and sorting of strings based on the collation specified in the schema. The choice of collation and collation version is up to the reader.
+- If the collation is not specified for a string type, then the reader must use the default utf8 binary collation.
 
-### Collations table feature
+## Writer Requirements for Collations:
+
+When Collations is supported (when the `writerFeatures` field of a table's protocol action contains `collations`), then:
```

Comment:
Do we require DomainMetadata to have at least one version that writers have to collect stats for and that readers can use?